AZ3166/tests/UnitTest/Sensor.ino

```diff
@@ -110,7 +93,18 @@ void accelerometer_test(){
     assertEqual(lsm6dsl->getXSensitivity(&data), RetVal_OK);
 }
 
-void gyroscope_test(){
+void gyroscope_test(){    
+    int axes[3];
+    float data;
+    DevI2C *i2c;
+    LSM6DSLSensor *lsm6dsl;
+    
+    i2c = new DevI2C(D14, D15);
```

Comment:
>i2c = new DevI2C(D14, D15); [](start = 4, length = 27)

Why not create a I2C instance globally instead of creating it every sonsor test function?

Reply:
Updated, set the I2C instance globally now.